e2e/credentials-server.e2e.spec.ts

```diff
@@ -67,11 +67,9 @@ describeWithCreds('Credentials server', () => {
   let authString: string;
   let page: Page;
   const appPath = path.join(__dirname, '../');
-  const command = os.platform() === 'win32' ? 'wsl' : 'curl';
-  // Assign these values on first request once we have an authString
-  // And we can't assign to ipaddr on Windows here because we need an async context.
-  let ipaddr: string|undefined = '';
-  let initialArgs: string[] = [];
+  const command = os.platform() === 'win32' ? 'curl.exe' : 'curl';
+  const initialArgs: string[] = []; // Assigned once we have auth string on first use.
+  const ipaddr = 'localhost';
```

Comment:
At this point we can drop `ipaddr` completely, right?

Reply:
I guess that's unnecessary future-proofing.